ui/pages/notifications/notifications.js

```diff
@@ -99,6 +99,7 @@ export default function Notifications() {
           type="secondary"
           className="notifications__header_button"
           onClick={markAllAsRead}
+          disabled={notifications.length === 0}
```

Comment:
Can you use `unreadNotifications` rather than `notifications` please. Otherwise the button will not be disabled if all the notifications have been read.

Reply:
Oh, great catch!

Reply:
@GuillaumeRx good suggestion! I've made this change.